Remove '$' from commands except where they are running in Darling.

hamishmb opened the pull request:
This is done for consistency in the remaining places that still use the '$' sign - a lot of the documentation doesn't do this any more.

Seeing as a few people have complained about it (#29 and #36), I figured this was worth doing.

CuriousTommy commented:
Other then what I stated above, it looks good to me! I like how you split out the command and output.

hamishmb commented:
What did you state above? I can't see it anywhere.

Yeah, I thought splitting them would improve readability.

CuriousTommy commented:
> What did you state above? I can't see it anywhere.

I'm referring to the review stuff that is above my comment.

hamishmb commented:
Hmm, it says there are no reviews at my end. I'll try again later on a different system and see if I can see from that.

CuriousTommy commented:
Sorry, I just realized that I didn't published my request.

hamishmb commented:
Got it now, thanks, and no problem. I'll update this when I get some time.

hamishmb commented on src/contributing/packaging.md:
````diff
@@ -7,27 +7,31 @@ To package Darling for Debian-based systems, we provide the `tools/debian/makede
 ### Install Dependencies
-```bash
-$ sudo apt install devscripts equivs dpkg-dev debhelper
+```
+bash
````
I changed this because many of the other code segments start with a new line - I was trying to make it consistent. Are you sure you want me to change these back?

CuriousTommy replied:
> I changed this because many of the other code segments start with a new line - I was trying to make it consistent.

Markdown supports a feature called [Syntax highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting).

---

What your change did was change this
```bash
$ sudo apt install devscripts equivs dpkg-dev debhelper
```
into this
```
bash
sudo apt install devscripts equivs dpkg-dev debhelper
```

It doesn't really make sense for the user to type in `bash` before they run the `sudo apt install` command.

hamishmb commented:
Okay, I think I've addressed the problems you found, and added the extra syntax highlighting correctly, as well as fixing some whitespace issues with the dependency lists, including the fix mentioned in #31.
